### DISCUSSION TITLE
Please give a way to exclude folder in "routes".

### DISCUSSION THREAD

**Discussion by Misaka-0x447f** [TanStack/router#2836] [General]
A library may shouldn't limit how people use that... If library developers wants to enforce something, it is okay to enforce by default but please **leave a switch there** -- not every project is ideal and people may need to keep their existing style in many reason, such as adding library to their existing project.

For now I have to given up using file based route, and expecting when I start my next project, there will be a switch.

**Comment by schiller-manuel** [comment]
please check the documentation

https://tanstack.com/router/latest/docs/framework/react/guide/file-based-routing/#options

![Screenshot 2024-11-23 at 22 11 24](https://github.com/user-attachments/assets/3303713b-9963-4da8-af5a-4317cb664f61)
